### ISSUE TITLE
Exec Sum and Appendix on Website

### ISSUE THREAD

**Issue by Isabelle1512** [18F/doi-extractives-data#1191]
The Exec Sum and the Appendix each had one typo. Corrected versions are ready to go on the website. Our designer also noticed that when you click "download" the Exec Sum, the document doesn't actually download, and the links don't work as a result.

[USEITI_Appendix_WEB_Version_FINAL_UPDATED_12_22.pdf](https://github.com/18F/doi-extractives-data/files/77375/USEITI_Appendix_WEB_Version_FINAL_UPDATED_12_22.pdf)
 
[USEITI_Report_Web_Version_Final_Updated_12_22.pdf](https://github.com/18F/doi-extractives-data/files/77404/USEITI_Report_Web_Version_Final_Updated_12_22.pdf)

**Comment by meiqimichelle** [comment]
Thanks for the issue, @Isabelle1512 ! We'll check on this and also update the files.

**Comment by meiqimichelle** [comment]
@Isabelle1512 I wasn't able to recreate the 'not download'. It does open in-browser, is that what your designer means? Also, I'm not sure which links don't work -- I tried a few, they seem OK.

Can you send me the url of the page with the broken executive summary link on it? I did find one link on the current live exec summary that doesn't work, but that looks like something encoded wrong in the pdf (ie, its not actually a link) (fyi, it seems to be in footnote 8 on pg 7).

So, in short, more details, please @Isabelle1512 :)

/cc @splatts so she can start getting used to how this works

**Comment by splatts** [comment]
@meiqimichelle @Isabelle1512   Thanks for the cc :) Good practice for me to see how this all works!

**Comment by Isabelle1512** [comment]
@meiqimichelle : I just tried it myself and had no issues...I'll follow up with Alyson, but it may just be something with her settings...

**Comment by Isabelle1512** [comment]
@meiqimichelle : Alyson says she will check again once the latest versions are up. I'm guessing it's fine.

**Comment by meiqimichelle** [comment]
@Isabelle1512 / @splatts -- the new files can be reviewed here:
https://federalist.18f.gov/preview/18F/doi-extractives-data/updates-exec-sum-files/about/report/

Re-assigning this issue to you for review. If it looks good, I will push this change to the live site.

**Comment by splatts** [comment]
@meiqimichelle / cc @Isabelle1512 Just reviewed. Everything looks good and those appear to be the correct versions of Exec Sum and Appendix. Typos are gone, and links are working. Thanks!

**Comment by meiqimichelle** [comment]
Great! I just pushed the changes to our development site, and I'll have you check this again on the live site when it merges. I'll ping you again.
